docs/Sonarr/sonarr-collection-of-custom-formats.md

```diff
@@ -44,44 +44,52 @@ We've made 3 guides related to this.
-|                       |                                         |                                         | [Quibi](#qibi)                              |
-|                       |                                         |                                         | [RTBF](#rtbf)                               |
-|                       |                                         |                                         | [SALTO](#salto)                             |
-|                       |                                         |                                         | [SHOWTIME](#sho)                            |
-|                       |                                         |                                         | [Stan](#stan)                               |
-|                       |                                         |                                         | [TVer](#tver)                               |
-|                       |                                         |                                         | [U-NEXT](#u-next)                           |
-|                       |                                         |                                         | [Videoland](#vdl)                           |
-|                       |                                         |                                         | [YouTube Red](#red)                         |
-|                       |                                         |                                         | [UHD Streaming Boost](#uhd-streaming-boost) |
-|                       |                                         |                                         | [UHD Streaming Cut](#uhd-streaming-cut)     |
-
-------
+---
+
+| Series Versions       | Unwanted                                | HQ Source Groups                        | Placeholder |
+| --------------------- | --------------------------------------- | --------------------------------------- | ----------- |
+| [Hybrid](#hybrid)     | [BR-DISK](#br-disk)                     | [Remux Tier 01](#remux-tier-01)         |             |
+| [Remaster](#remaster) | [Extras](#extras)                       | [Remux Tier 02](#remux-tier-02)         |             |
+|                       | [LQ](#lq)                               | [HD Bluray Tier 01](#hd-bluray-tier-01) |             |
+|                       | [LQ (Release Title)](#lq-release-title) | [HD Bluray Tier 02](#hd-bluray-tier-02) |             |
+|                       | [Upscaled](#upscaled)                   | [WEB Tier 01](#web-tier-01)             |             |
+|                       | [x265 (HD)](#x265-hd)                   | [WEB Tier 02](#web-tier-02)             |             |
+|                       |                                         | [WEB Tier 03](#web-tier-03)             |             |
+|                       |                                         | [WEB Scene](#web-scene)                 |             |
```

Comment:
Placeholder column intentional?

Reply:
for the time being it's intentional, and makes the tables the same size